index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+    </style>
+</head>
+<body>
+
+    <header>
+        <div class="nav-menu-content">
+
+            <div class="logo"><a class=mylogo> My Logo </a></div>
+
+            <nav>
+                <ul class="nav-menu">
+                    <li class="nav-items"><a href="#">Portfolio</a></li>
+                    <li class="nav-items"><a href="#">About</a></li>
+                    <li class="nav-items"><a href="#">Contact</a></li>
+                  </ul>
+
```

Comment:
- [ ] [REQUIRED] Please make sure that you use the same font, spacing, and color specified in the design.

Your Implementation | Figma Design
--------------------|--------------
![](https://user-images.githubusercontent.com/59416136/154070487-d80badd7-107e-4efe-80c0-d46a117af966.png) | ![](https://user-images.githubusercontent.com/59416136/154070707-18c5f3fd-ed17-40a8-87b7-4dce79d53ca3.png)

Reply:
![image](https://user-images.githubusercontent.com/87770050/164404941-3e5d9db3-777c-41b8-bdb8-4085bb573a39.png)

Reply:
as you can see in this screenshot, when I open the same file I am not seeing the same outcome, I am not seeing alignment issues or background overlapping issues, I have checked that all my commits are up to date and have checked and opened everything multiple times and every time I get this same page, I do not know why you received a different page, I even played around withis screen size and I have no issues, why are you getting a different outcome to me?